doc/source/rllib-toc.rst

```diff
@@ -124,7 +124,7 @@ Algorithms
 
    -  |pytorch| :ref:`Model-Based Meta-Policy-Optimization (MBMPO) <mbmpo>`
 
-   -  |pytorch| :ref:`Dreamer <dreamer>`
+   -  |pytorch| :ref:`Dreamer (DREAMER) <dreamer>`
```

Comment:
Dreamer isn't a meta RL algo. It is a model RL algo for vision.

Reply:
Let me actually rename this section to MB / ML and add AlphaZero